app/build.gradle

```diff
@@ -23,6 +23,11 @@ dependencies {
     implementation project(':etor')
     testImplementation testFixtures(project(':shared'))
 
+    // Patch: Add the fixed Jetty version
+    // Import the Jetty BOM to pull consistent Jetty versions
+    implementation 'org.eclipse.jetty:jetty-bom:12.0.14'
```

Comment:
Javalin pulls a bill of materials so I'm doing the same in order to have all dependencies working correctly

Reply:
Just a contextual question since I'm not super familiar with Gradle - how do you test this? Like if you build locally, can you see what version of Jetty is getting used?

Reply:
Can we remove `org.eclipse.jetty:jetty-bom:12.0.14` once javalin is updated with the patch? Or is the intention to leave it?

Reply:
> Just a contextual question since I'm not super familiar with Gradle - how do you test this? Like if you build locally, can you see what version of Jetty is getting used?

You can run the following to see how Gradle resolved all the dependencies versions.
`./gradlew dependencies`

> Can we remove `org.eclipse.jetty:jetty-bom:12.0.14` once javalin is updated with the patch? Or is the intention to leave it?

Yes, normally you would remove it.